aloscene/frame.py

```diff
@@ -350,6 +350,22 @@ def append_scene_flow(self, scene_flow: SceneFlow, name: Union[str, None] = None
         """
         self._append_child("scene_flow", scene_flow, name)
 
+    def as_numpy(self, dtype=torch.uint8):
```

Comment:
Thanks @eleonorefrcs for reopening.
As the method `as_numpy` is already in AugTensors and used in many lines of our code, overriding it is not a good idea. I suggest to rename the function to `as_image`.